### PR TITLE
Fixed PS-7917 (TokuDB storage engine can't be installed with ps-admin)

### PR DESCRIPTION
**Pull request by altmannmarcelo**
https://jira.percona.com/browse/PS-7917

Problem
New tokudb deprecation error was logging a message before logging
service for plugins was initialized.

Fix
Moved the deprecation error to after logging service initialization.

Adjusted error message to display loose-X configuration. Otherwise,
server will fail to start if tokudb_enabled or tokudb_backup_enabled are
specified but plugin is not loaded.

Adjusted error message to not mention that those variables require a
server restart. They will be read at plugin initialization, thus not
requiring a complete server restart.

Added check at ps-admin to also display the deprecation error message.

### CODE REVIEW

**Comment by altmannmarcelo** [comment]
```
./mtr --suites=tokudb,tokudb_add_index,tokudb_alter_table,tokudb_backup,tokudb_bugs,tokudb_deprecation,tokudb_parts,tokudb_perfschema,tokudb_rpl,tokudb_sys_vars --parallel=28 --big-test --force --max-test-fail=0

. . .

Completed: Failed 48/609 tests, 92.12% were successful.

Failing test(s): tokudb_bugs.db739_replace tokudb_parts.partition_alter4_tokudb tokudb.hotindex-del-0 tokudb.rows-32m-seq-insert tokudb_deprecation.tokudb_backup_explicitly_enabled tokudb_bugs.db739_insert tokudb_backup.rpl_tokudb_commit_sync tokudb_sys_vars.tokudb_analyze_in_background_basic tokudb.fast_update_blobs tokudb_
recation.tokudb_backup_default_settings tokudb.change_column_char tokudb.bf_insert_select tokudb_parts.partition_max_parts_hash_tokudb tokudb_parts.partition_max_sub_parts_key_range_tokudb tokudb_parts.part_supported_sql_func_tokudb tokudb_bugs.5733_innodb tokudb.hotindex-del-1 tokudb_parts.partition_max_sub_parts_list_tokudb tokudb.change_column_all_1000_1 tokudb_alter_table.hcad_fixedvar_add2 tokudb.change_column_all_1000_10 tokudb_parts.partition_max_sub_parts_key_list_tokudb tokudb_backup.innodb_use_native_aio_enabled tokudb_backup.tokudb_backup_set_last_error tokudb_rpl.rpl_parallel_tokudb tokudb_backup.backup_master_info tokudb_alter_table.hcad_all_add shutdown_report tokudb_alter_table.hcad_all_fixed_add tokudb_backup.tokudb_backup_exclude tokudb.fast_update_blobs_with_varchar tokudb_parts.partition_max_parts_key_tokudb tokudb_parts.partition_max_parts_list_tokudb tokudb_backup.empty_slave_info_file tokudb_backup.backup_master_state tokudb.change_column_int_not_supported tokudb.change_column_bin tokudb_parts.partition_alter1_1_2_tokudb tokudb_rpl.rpl_tokudb_bug28430 tokudb.hotindex-update-1 tokudb_alter_table.hcad_all_add2 tokudb.hotindex-update-0 tokudb_parts.partition_max_parts_range_tokudb tokudb.type_temporal_fractional tokudb_parts.partition_max_sub_parts_range_tokudb tokudb_bugs.5733_tokudb
```

File: scripts/ps-admin.sh [hunk 1]
```diff
@@ -319,6 +326,13 @@ fi
 
 # Check if TokuDB plugin available on the system
 if [ ${ENABLE_TOKUDB} = 1 ]; then
+  # Warn about TokuDB deprecation
+  echo "${MYSQL_DEFAULTS_BIN} mysqld ${DEFAULTS_FILE_OPTION}| grep -v '#' | grep -c tokudb_enabled"
+  STATUS_ENABLE_TOKUDB_MYCNF=$(${MYSQL_DEFAULTS_BIN} mysqld ${DEFAULTS_FILE_OPTION}| grep -v '#' | grep -c tokudb_enabled)
+  if [ ${STATUS_ENABLE_TOKUDB_MYCNF} = 0 ]; then
```
Comment (george-lorch) [inline scripts/ps-admin.sh:331]: Is this 0 only if tokudb_enabled=0 or is this also 0 if the tokudb_enabled is not present in the file at all?

Comment (altmannmarcelo) [reply scripts/ps-admin.sh:331]: That is a good point, this is only validating if the config is not present in the file. Need to validate for present and <> 1

Comment (altmannmarcelo) [reply scripts/ps-admin.sh:331]: @georgelorchpercona fixed. Now we count the occurrences of  =1 | true and make sure it appears at least once:

```
STATUS_ENABLE_TOKUDB_MYCNF=$(${MYSQL_DEFAULTS_BIN} mysqld ${DEFAULTS_FILE_OPTION}| grep -v '#' | grep tokudb_enabled | egrep -ic '=1|=true')
```

**Review by george-lorch — APPROVED** [review]

**Pull request merged**